odoo/addons/base/models/ir_attachment.py

```diff
@@ -697,3 +697,23 @@ def action_get(self):
     def _get_serve_attachment(self, url, extra_domain=None, order=None):
         domain = [('type', '=', 'binary'), ('url', '=', url)] + (extra_domain or [])
         return self.search(domain, order=order, limit=1)
+
+
+class Model(models.AbstractModel):
+    _inherit = 'base'
+
+    linked_attachment_ids = fields.One2many(
+        comodel_name='ir.attachment',
+        compute='_compute_linked_attachment_ids',
+        groups="base.group_user",
+        automatic=True,
+    )
```

Comment:
Please explain the presence of `groups` and `automatic`.  What is their added value?
Please also justify polluting the field space of _all_ models for optimizing _one_ model.
It really looks like you are tweaking the framework for the accounting modules' needs.

Reply:
The `groups` is basically because of this (and other similar access rights checks); it would raise errors in some tests because the field would be displayed otherwise, without enough rights to do so.
https://github.com/odoo/odoo/blob/e4a09ce62e165b8a33797702f89695debee15968/odoo/addons/base/models/ir_attachment.py#L445-L446

I will push a version without the `automatic`, but as I understand it, it is there for the same reasons as for `display_name`

Reply:
I have pushed a version without those attributes set, here is the result:
https://runbot.odoo.com/runbot/batch/1062568/build/41373652